README.md

```diff
@@ -84,10 +84,6 @@ This repository's contents is divided across four primary sections:
 
 - [`/infra/terraform`](infra/terraform): contains Terraform modules for deploying HASH on AWS
 
-### Libraries
```

Comment:
Removing this will leave a deadlink for the following lines:

https://github.com/hashintel/hash/blob/8539c0ae7760e19ee39a201b0c9a33fa9434ca4e/README.md?plain=1#L62

Reply:
Good spot, thanks! It should be fixed with e1f3a5fddcf4ebb945428b4405cd3ed65d1d3a27